picking_good_slos.md

```diff
@@ -0,0 +1,57 @@
+Each stakeholder understands the outcomes of both meeting or missing an SLO. 
+
+So with that in mind, what should be our service level objective? 
+
+A new service presents the opportunity to use an SLO to steer development efforts before reaching production. 
+Ideally, a new service in production is both featureful and reliable. Product managers responsible for the service could use an SLO to define the production readiness by its limited and general availability or other aspects of the service. The availability of a service typically has deadlines, and an SLO can assist in making a clear decision about the risk for a service.
+
+Do we focus our time on making the service more reliable or adding a feature that customer X wanted?
+
+The answer to the above is your decision to make - SLO is the tool used to remove subjectivity about components of the decision. 
+
+To learn more about the responsibilities across different personas in your organization, read our [article](./ADRs/RH/SIG-SRE/ADR-00006%20SLO%20RACI%20Chart.md) on the RACI model
+
+## Case 2 - Existing Service 
+
+For this scenario, imagine we created our SLIs for an existing Authentication service in production and defined an SLO. Suddenly, one day, we get paged that our SLO exceeded our target level.
+After a quick investigation, we notice that our 3rd party auth provider is down. Consequently, our auth service is in degraded mode. 
+
+Since we have exceeded our SLO because of a third-party service, should we stop releasing new features?
+
+When creating an SLO, external dependencies should be considered, and mitigation strategies should be employed to handle service disruptions from those external dependencies. 
+Once your organization is convinced of the benefits that SLOs bring across multiple teams, retrofitting this into a mature service has some cultural considerations. 
+
+First, all stakeholders have to agree on the outcomes of missing an SLO [burning the error budget](./ADRs/RH/SIG-SRE/ADR-00007-Error-Budget-Policy.md). 
+This decision may take time for engineering teams to digest the idea of slowing feature work and focusing on reliability issues. 
+The goal here should be the balance of direct impact on current processes vs introducing new processes. 
+
+There are two main paths this could be navigated, and each should consider if there is pre-existing data that provides a baseline of current service levels. 
+* Set a high SLO to start with and work towards it. What is high? 99.9%? This decision depends on if you can approximate the current service levels being delivered. 
+* Set a low SLO to be more tight interally and catch potential issues earlier. However, tight SLOs lead to higher alerting frequency. 
+
+# Tips
+* Reflect on customer experience: Customer satisfaction drives the sustainability of the business, so make sure to focus on critical paths that make customers happy.
```

Comment:
Number one tip is "having good SLIs based on what is important to customers". :)

Reply:
I believe this is referred multiple times to our SLI guide and wanted to focus only on SLO stuff and let the reader follow the "Picking good SLIs" guide for more SLI tips